plugin/src/test/fixtures/sources/ktlint/with-error/KtLintViolator.kt

```diff
@@ -0,0 +1,7 @@
+class KtLintViolator {
+
+    override fun equals(other: Any?): Boolean {
+        // this is not allowed
```

Comment:
What is not allowed?

Reply:
Haha I think always returning true is not allowed. This is just a fixture and a copy paste from detekt fixtures.

Reply:
Here, I need any Kotlin code with style problem. Would be happy to change. 👍